tests/test_core.py

```diff
@@ -7709,6 +7709,89 @@ def test_emscripten_scan_registers(self):
     self.set_setting('ASYNCIFY', 1)
     self.do_run_in_out_file_test('tests', 'core', 'emscripten_scan_registers')
 
+  @no_fastcomp('wasm-backend specific feature')
+  @no_wasm2js('TODO: lazy loading in wasm2js')
+  @parameterized({
+    'conditional': (True,),
+    'unconditional': (False,),
+  })
+  def test_emscripten_lazy_load_code(self, conditional):
```

Comment:
How about just "test_lazy_load_code"?

Reply:
It's the name of the C function, though?

Reply:
OK SGTM either way.     I was think thinking "lazy_load_code" is kind of concept you are testing since its also the suffix of `ASYNCIFY_LAZY_LOAD_CODE`.